assets/app/view/game/dividend.rb

```diff
@@ -52,6 +57,9 @@ def render
             margin: '0.5rem 0 0 0',
             textAlign: 'left',
           },
+          hook: {
```

Comment:
in order to have a hook you need a key, look at view/chat.rb as an example

Reply:
Is the hook needed?
I don't know what it does. I just copy-pasted code from route_selector.rb

Reply:
i don't know, what happens when you remove it?

Reply:
lol

It works without the hook on the cases I tried. But I assume it was there because maybe some combination of clicks may remove the view, but not advance the action in the "normal way". Then the function `cleanup` would not be called and the storage of the `:routes` thingy might conflict with something else somewhere else.

Reply:
I went back and found out that you added the hook in this commit: https://github.com/tobymao/18xx/commit/cfe18c198fe0f014356bea8c54f080ec439ff4fd#diff-c0fbc2481cab263a628acff8e00c30eb

So my guess is that it would cause problems with the Undo.
Can I just put whatever string I want as key? If so, I already edited the PR.